### ISSUE TITLE
write error messages for scpi_control.searcher.searchDevices

### ISSUE THREAD

**Issue by ilka-schulz** [Zahner-elektrik/zahner_potentiostat#7]
I am calling `zahner_potentiostat.scpi_control.searcher.searchDevices()` with a `Zennium X` connected to my computer via USB which has a `PP242` on channel `1` of its `EPC42`. The function runs for very long (sth. like 20 s) and I get the following output in my console:

```text
Serial interfaces found: ['/dev/ttyUSB2', '/dev/ttyUSB1', '/dev/ttyUSB0', '/dev/tty7']
/dev/ttyUSB2:
error: /dev/ttyUSB2
/dev/ttyUSB1:
IDN?

error: /dev/ttyUSB1
/dev/ttyUSB0:
error: /dev/ttyUSB0
/dev/tty7:
error: /dev/tty7
```

I know, I am trying to detect a Zennium device via this library but I think Zennium devices are SCPI devices as well, so it should show up. Even if it should not work, the function should not take that long and it should print some readable error messages

When I connect the `PP242` to my computer directly via USB, the function behaves as expected within reasonable time.

The error message `error` is way to short to properly debug the connection. Please add some error messages, it would help users a lot with debugging.

**Comment by maxkrapp1** [comment]
The Zennium X needs Thales as runtime software on Windows and is not compatible with this library.

You are also welcome to contact me personally.
maximilian.krapp@zahner.de

**Comment by maxkrapp1** [comment]
This will be fixed in the next release.

The error messages will be more meaningful and with a process pool with a maximum of 4 processes, the search process will be speeded up significantly.

**Comment by maxkrapp1** [comment]
Fixed with commit 6d210947ca02cc7607018df6982153eb1c86258f in version 1.1.0.

**Comment by ilka-schulz** [comment]
> Fixed with commit [6d21094](https://github.com/Zahner-elektrik/zahner_potentiostat/commit/6d210947ca02cc7607018df6982153eb1c86258f) in version 1.1.0.

I think you mentioned a wrong commit? Anyways, I am happy with the current error messages, no need to reopen the issue.
